DevelopGitHub/GitHubExtension/GitHubExtension.Statistics.WebApi/Queries/Implementations/StatisticsQuery.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Threading.Tasks;
+using System.Web;
+using GitHubExtension.Statistics.WebApi.CommunicationModels;
+using GitHubExtension.Statistics.WebApi.Extensions.Cookie;
+using GitHubExtension.Statistics.WebApi.Mappers.GitHub;
+using GitHubExtension.Statistics.WebApi.Queries.Interfaces;
+
+namespace GitHubExtension.Statistics.WebApi.Queries.Implementations
+{
+    public class StatisticsQuery : IStatisticsQuery
+    {
+        #region fields
+        private readonly IGitHubQuery _gitHubQuery;
+        private DateTime timeTo = DateTime.Now;
+        private DateTime timeFrom;
+        private int countDaysInYear = 364;
+        private int countMounthInYear = 12;
+        private ICollection<string> _activityMonths;
+        private ICollection<RepositoryModel> _repositories;
+        private ICollection<ICollection<int>> _commitsRepositories;
+        private ICollection<int> _commitsForYear;
+        private ICollection<int> _commitsRepository;
+        private int countFollower;
```

Comment:
Some these variables is global variables, and i relocated their to top of class.
But I did a mistake when I relocated some variables to top of class i relocated some local variables.
I know it's very bad. And i'm going to fix it .